dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/StaticVarCompensator/Parameters.mo

```diff
@@ -0,0 +1,54 @@
+within Dynawo.Electrical.Controls.Voltage.StaticVarCompensator;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+package Parameters "Parameters of the static var compensator"
+
+  record Params_Regulation
+    import Dynawo.Types;
+    parameter Types.ApparentPowerModule SNom "Static Var Compensator nominal apparent power in MVA";
+    parameter Types.PerUnit Lambda "Statism of the regulation law URefPu = UPu + Lambda*QPu in p.u (base UNom, SNom)";
+    parameter Types.PerUnit KG "Global gain of the regulation";
+    parameter Types.PerUnit Kp "Proportional gain of the PI controller";
+    parameter Types.Time Ti "Integral time constant of the PI controller";
+  end Params_Regulation;
+
+  record Params_Limitations
+      import Dynawo.Types;
+      parameter Types.PerUnit BMaxPu "Maximum value for the variable susceptance in p.u (base SNom)";
+      parameter Types.PerUnit BMinPu "Minimum value for the variable susceptance in p.u (base SNom)";
+      parameter Types.PerUnit IMaxPu "Maximum value for the current in p.u (base UNom, SNom)";
+      parameter Types.PerUnit IMinPu "Minimum value for the current in p.u (base UNom, SNom)";
+      parameter Types.PerUnit KCurrentLimiter "Integral gain of current limiter";
+  end Params_Limitations;
+
+  record Params_CalculBG
+    import Dynawo.Types;
+    parameter Types.PerUnit BShuntPu "Fixed susceptance of the static var compensator in p.u (for standby mode) (base SNom)";
+  end Params_CalculBG;
+
+  record Params_ModeHandling
+    import Dynawo.Types;
+    parameter Types.VoltageModule UNom "Static var compensator nominal voltage in kV";
+    parameter Types.VoltageModule UBlock "Voltage value under which the static var compensator is blocked";
+    parameter Types.VoltageModule UDeblockUp "Upper voltage value defining the voltage interval in which the static var compensator is deblocked";
+    parameter Types.VoltageModule UDeblockDown "Lower voltage value defining the voltage interval in which the static var compensator is deblocked";
```

Comment:
I'm not very sure of Deblock. Certainly unblock but it leads to an uggly name UUnblockUp

Reply:
fixed